examples/userdemo/anchored_box01.py

```diff
@@ -10,8 +10,9 @@
 
 fig, ax = plt.subplots(figsize=(3, 3))
 
-at = AnchoredText("Figure 1a",
-                  prop=dict(size=15), frameon=True, loc=2)
+# loc=8 is equivalent to loc='lower center'
+at = AnchoredText("Here is some\nsample text",
```

Comment:
@ImportanceOfBeingErnest @jklymak Would this be considered an acceptable change?
1. Modified the string to have a newline, because that (A) makes it less redundant, and (B) shows off the capabilities a little better
2. Changed (and documented) the `loc` parameter.